arxiv/taxonomy.py

```diff
@@ -0,0 +1,1479 @@
+"""arXiv group, archive and category definitions."""
+
+GROUPS = {
+    'grp_physics': {
+        'name': 'Physics',
+        'start_year': 1991,
+        'default_archive': 'hep-th'
+    },
+    'grp_math': {
+        'name': 'Mathematics',
+        'start_year': 1992,
+        'default_archive': 'math'
+    },
+    'grp_q-bio': {
+        'name': 'Quantitative Biology',
+        'start_year': 1992,
+        'default_archive': 'q-bio'
+    },
+    'grp_cs': {
+        'name': 'Computer Science',
+        'start_year': 1993,
+        'default_archive': 'cs'
+    },
+    'grp_test': {
+        'name': 'Test',
+        'start_year': 1995,
+        'is_test': True
+    },
+    'grp_q-fin': {
+        'name': 'Quantitative Finance',
+        'start_year': 1997
+    },
+    'grp_stat': {
+        'name': 'Statistics',
+        'start_year': 1999
+    },
+    'grp_eess': {
+        'name': 'Electrical Engineering and Systems Science',
+        'start_year': 2017
+    },
+    'grp_econ': {
+        'name': 'Economics',
+        'start_year': 2017
+    }
+}
+DEFAULT_GROUP = 'physics'
+
+ARCHIVES = {
+    'acc-phys': {
+        'name': 'Accelerator Physics',
+        'in_group': 'grp_physics',
+        'start_date': '1994-11',
+        'end_date': '1996-09',
+    },
+    'adap-org': {
+        'name': 'Adaptation, Noise, and Self-Organizing Systems',
+        'in_group': 'grp_physics',
+        'start_date': '1993-03',
```

Comment:
Nit-picky, but any reason not to use ``datetime.date``s for these?

Reply:
Sounds good to me, will update